docs/platform/tutorials/image-processing/other-services.md

```diff
@@ -0,0 +1,195 @@
+# Other services
+
+There are some additional services in the pipeline that provide useful functionality. These range from S3 storage of data to calculation of the maximum vehicles per day in a specific location.
+
+![Other services](./images/other-services-pipeline-segment.png)
+
+Briefly, these services are:
+
+1. *Cam vehicles* - calculates the total vehicles, where vehicle is defined as one of: car, 'bus', 'truck', 'motorbike'. This number is fed into the *Max vehicle window* service.
+
+2. *Max vehicle window* - calculates the maximum vehicles over a time window of one day. This service sends messages to the Data API service.
+
+3. *Data API* - this REST API service provide two endpoints: one returns the *Max vehicle window* values for the specified camera, and the other endpoint returns camera data for the specified camera. This API is called by the UI to obtain useful data.
+
+4. *S3* - stores objects in Amazon Web Services (AWS) S3. This service enables you to persist any data or results you might like to keep more permanently.
+
+## Cam vehicles
```

Comment:
Might be prudent to mention that the stream-id acts as the primary key and is still available.... even though it's not in the message body. For context: I was trying to write  an aggregation challenge, where you only keep the latest maximums for each camera, but was confused because the camera id is not in this message example.... but then I realized it is still accessible somehow but passed as metadata that is not displayed here.

Reply:
Definitely worth making that a note. Stream id is available as metadata as you say `stream_id = stream_consumer.stream_id`.

Reply:
Fixed.